src/MCPServer/lib/unit.py

```diff
@@ -34,7 +78,30 @@ def setMagicLink(self,link, exitStatus=""):
         return
 
     def setVariable(self, variable, variableValue, microServiceChainLink):
-        return
+        if not variableValue:
+            variableValue = ""
```

Comment:
I think it's valid to have an empty variable value, but not valid to have a variable with no microServiceChainLink. I see `null=True` in the model definition, but I'm not sure the original code allowed that?